e2e-test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
This file works on `bash`, but it doesn't work with all shell (e.g. `fish`) and I have some issue with `pkill -t $(ps hotty $$)` command, that doesn't kill meteor server. It is also a quite complex script, not easy to mantain.

What do you think about rewrite that as node script?